src/app/features/vendor/vendor-listing/listing.component.ts

```diff
@@ -1,10 +1,99 @@
-import { Component } from '@angular/core';
-//import { CommonModule } from '@angular/common';
+import { Component,OnInit } from '@angular/core';
+import { Vendor } from '../models/vendor.model';
+
+// LLD
```

Comment:
Please refer the sample LLD and follow the same pattern

Reply:
LLD updated